app/serializers/basic_reviewable_flagged_post_serializer.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class BasicReviewableFlaggedPostSerializer < BasicReviewableSerializer
+  attributes :post_number, :topic_title
+
+  def post_number
+    object.post.post_number
+  end
+
+  def topic_title
+    object.topic.fancy_title
+  end
+
+  def include_post_number?
+    object.post.present?
```

Comment:
Just to confirm, is the post preloaded as part of `target`?

Reply:
Yes, the `Reviewable.recent_list_with_pending_first` has an `includes` for a bunch of things including `target`.